src/main/java/org/spongepowered/api/service/economy/EconomyService.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.service.economy;
+
+import com.google.common.base.Optional;
+import org.spongepowered.api.entity.player.Player;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface EconomyService {
+    /**
+     * Retrieves the {@link Account} denoted by the
+     * given owner string.
+     * 
+     * <p>If the Account does not exist, this method will
+     * return Optional.absent().</p>
+     * 
+     * <p>Owner strings are not representative of players
+     * only. See {@link Account#getOwner()} for
+     * reference.</p>
+     * 
+     * @param owner the owner string to search with
+     * 
+     * @return the Account owned by the given owner, or
+     *         Optional.absent() otherwise
+     * 
+     * @see Account#getOwner()
+     * @see EconomyService#createNewAccount(String)
+     */
+    public Optional<Account> getAccount(String owner);
+    
+    /**
+     * Creates a new {@link Account} and assigns it to the
+     * given owner string.
+     * 
+     * <p>This method cannot override existing Accounts. If
+     * the Account already exists, this method will return
+     * false to indicate that creation has failed. To
+     * override an existing Account, it should be reset to
+     * be deleted before calling this method upon it.</p>
+     * 
+     * <p>Owner strings are not representative of players
+     * only. See {@link Account#getOwner()} for
+     * reference.</p>
+     * 
+     * @param owner the owner string to assign the new
+     *        Account to
+     * 
+     * @return whether or not the Account was created
+     *         successfully
+     */
+    public boolean createNewAccount(String owner);
```

Comment:
Should return the account. If it may fail, return `Optional<Account>`
Also, methods in interface don't need the `public` access modifier declared explicitly.